customization/parallel-data.md

```diff
@@ -30,16 +36,16 @@ Parallel data can have errors, like misaligned sentences, bad sentence segmentat
 
 | Name | Type |
 | ---- | ---- |
-| CCAligned | Data repository |
-| CCMatrix | Data repository |
-| Clarin | Data repository |
-| Europarl | Data set |
-| FLORES | Data set |
-| Hansard | Data set |
-| JESC | Data set |
-| Mozilla | Data repository |
-| OpenSubtitles | Data repository |
-| ParaCrawl | Data repository |
-| VoxPopuli | Data set |
-| WikiMatrix | Data repository |
-| WikiTitles | Data repository |
+| [CCAligned](https://opus.nlpl.eu/CCAligned.php) | Data repository |
+| [CCMatrix](https://github.com/facebookresearch/LASER/tree/main/tasks/CCMatrix) | Data repository |
```

Comment:
cc @cefoo 
IMHO, CCMatrix, WikiMatrix and Mozilla Common Voice are datasets, not repositories.

Repositories are more like collections that just point to datasets created by somebody else and often hosted somewhere else.

For example, this page.